pkgs/applications/networking/instant-messengers/jami/daemon.nix

```diff
@@ -57,9 +58,19 @@ let
 
   pjsip-jami = pjsip.overrideAttrs (old:
     let
+      version = "f94b3d72b428d7c59b319e35095de711ed1d8534";
       patch-src = src + "/daemon/contrib/src/pjproject/";
     in
     {
+      inherit version;
+
+      src = fetchFromGitHub {
+        owner = "savoirfairelinux";
+        repo = "pjproject";
+        rev = version;
+        sha256 = "0skz6ld4w1nriw4khf05f0yh42mfdc9g28fbbnkxlk2qrdd766ac";
```

Comment:
Does the update script updates this hash as well?

Reply:
No. I don't know how to do that.